packages/trader/src/Modules/Trading/Containers/trade.tsx

```diff
@@ -92,6 +94,28 @@ const Trade = observer(() => {
         return null;
     }, [try_synthetic_indices, try_open_markets, category, subcategory]);
 
+    React.useEffect(() => {
+        const html = document.querySelector('html');
+        if (isTabletPortrait && isTabletOs) {
+            setShouldShowPortraitLoader(true);
+            if (!is_chart_loading && !should_show_active_symbols_loading) {
+                rotateTimeout.current = setTimeout(() => {
+                    html?.classList.add('tablet-landscape');
+                    setTimeout(() => {
```

Comment:
Do we need to clear second timeout? Please, ignore if irrelevant

Reply:
@balakrishna-deriv suggested to hide the loader after the rotation is done only. So we need this extra timeout

Reply:
I mean there is no way, that second (inner) timer will be set if we clear first one (external), right?